src/Compilers/CSharp/Portable/Lowering/LocalRewriter/LocalRewriter_AssignmentOperator.cs

```diff
@@ -320,8 +320,7 @@ private BoundExpression VisitAssignmentOperator(BoundAssignmentOperator node, bo
                 argsToParamsOpt,
                 ref argumentRefKindsOpt,
                 out argTemps,
-                invokedAsExtensionMethod: false,
-                enableCallerInfo: ThreeState.True);
```

Comment:
I'm not sure why the diff shows a change related to `enableCallerInfo` argument, here and in some other local rewriter files. Are those intentional changes?